code/model/modelrender.h

```diff
@@ -74,6 +75,9 @@ class model_render_params
 	int Insignia_bitmap;
 
 	int *Replacement_textures;
+	bool manage_replacement_textures; // This is set when we are rendering a model without an associated ship object;
```

Comment:
As far as I can tell (and the compiler backs me up) there's only one place in the codebase where a model_render_params object was actually copied. I've changed that and disabled the copy operator.